xcode/Subconscious/Shared/Library/SQLite3Database.swift

```diff
@@ -70,30 +70,10 @@ final class SQLite3Database {
                 return nil
             }
         }
-
-        public func get(_ i: Int) -> String? {
```

Comment:
Removing these deprecated accessors. They were overloaded on return type, which is discouraged in Swift, because the Swift type checker is bad at deducing correct code path from return type.